lib/assessment.js

```diff
@@ -8,24 +8,49 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-var min = function min() {
+var min = function min(previousValue, currentValue, index, array) {
+
+  var minValue = previousValue;
+
+    if(currentValue < minValue) {
+      minValue = currentValue;
+    }
+  return minValue;
+  }
```

Comment:
Very nice! 

If you wanted to be very terse, you could also write it as follows:
```javascript
var min = function(a, b) {
  return (a < b)? a : b;
};
```